.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: php
 php:
-- 5.5
```

Comment:
Are you setting the base requirement for 5.5 or 5.6. If you still want to _support_ 5.5, you should probably leave the tests on.